OpenRA.Mods.Common/Activities/Air/FallToEarth.cs

```diff
@@ -20,16 +20,14 @@ public class FallToEarth : Activity
 	{
 		readonly Aircraft aircraft;
 		readonly FallsToEarthInfo info;
-		int acceleration = 0;
-		int spin = 0;
+
+		int spin;
 
 		public FallToEarth(Actor self, FallsToEarthInfo info)
 		{
 			this.info = info;
 			IsInterruptible = false;
 			aircraft = self.Trait<Aircraft>();
-			if (info.Spins)
-				acceleration = self.World.SharedRandom.Next(2) * 2 - 1;
```

Comment:
I just noticed that this is removing the random spin direction - all units will now spin anti-clockwise.